ntbs-service/Models/MDRDetails.cs

```diff
@@ -0,0 +1,27 @@
+using Microsoft.EntityFrameworkCore;
+using System.ComponentModel.DataAnnotations;
+using ntbs_service.Models.Enums;
+using ExpressiveAnnotations.Attributes;
+using ntbs_service.Models.Validations;
+
+namespace ntbs_service.Models
+{
+    [Owned]
+    public class MDRDetails : ModelBase
+    {
+        public Status? ExposureToKnownCaseStatus { get; set; }
+
+        [MaxLength(40)]
+        [RegularExpression(ValidationRegexes.CharacterValidation, ErrorMessage = ValidationMessages.StandardStringFormat)]
+        [RequiredIf(@"ExposureToKnownCaseStatus == Enums.Status.Yes", ErrorMessage = ValidationMessages.RelationshipToCaseIsRequired)]
+        public string RelationshipToCase { get; set; }
+
+        [RequiredIf(@"ExposureToKnownCaseStatus == Enums.Status.Yes", ErrorMessage = ValidationMessages.CaseInUKStatusIsRequired)]
+        public Status? CaseInUKStatus { get; set; }
+
+        public int? RelatedNotificationId { get; set; }

```

Comment:
I initially tried adding this as a foreign key, but EF core kept getting confused because there is also a relationship to the owner's notificationId and it wasn't clear how to fix this... so I decided that we should just keep it as a normal property and fetch the notification when required.

Reply:
Fair enough! There must be a way to tell it to EF specifically, but it shouldn't really impact us massively - do leave a comment in the code explaining the lack and the reason please :)

Reply:
Are we missing out on ef making this a FK for us though?
If so, we should add it manually I reckon

Reply:
@jasiekmiko So the problem is that as soon as I make it a Foreign Key, the created migration tries to delete the foreign key to the owner notification. I also tried just modifying the migration, but then when I was seeding data (to populate the owned model) there was another error (can't quite remember).
I think somehow EF core is confused when you have an owner and a foreign key with the same type as the owner. I couldn't really find anything about it, and didn't want to spend ages looking considering we don't necessarily need this to be FK relationship - I left a comment now, so maybe we can fix at a different time. Should I add a tech debt ticket?

Reply:
I think we're going to have to live with this for now then!
The biggest downside is that it *could* get out of sync, at least in theory. so A tech debt ticket seems worth it, even if it might not become a high-priority one!